healthcare/api-client/v1/datasets/datasets.py

```diff
@@ -0,0 +1,450 @@
+# Copyright 2018 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+
+from googleapiclient import discovery
+from googleapiclient.errors import HttpError
+from google.oauth2 import service_account
+
+
+# [START healthcare_get_client]
+def get_client(service_account_json):
+    """Returns an authorized API client by discovering the Healthcare API and
+    creating a service object using the service account credentials JSON."""
+    api_scopes = ['https://www.googleapis.com/auth/cloud-platform']
+    api_version = 'v1'
+    discovery_api = 'https://healthcare.googleapis.com/$discovery/rest'
+    service_name = 'healthcare'
+
+    credentials = service_account.Credentials.from_service_account_file(
+        service_account_json)
+    scoped_credentials = credentials.with_scopes(api_scopes)
+
+    discovery_url = '{}?version={}'.format(
+        discovery_api, api_version)
```

Comment:
It looks like this API is publicly listed at https://www.googleapis.com/discovery/v1/apis, so you shouldn't need a `discovery_url`.

Providing the service name and version should be sufficient for building the client.

```py
discovery.build(service_name, api_version, credentials=scoped_credentials)
```

Reply:
Done in all files.